app/models/ext_management_system.rb

```diff
@@ -186,6 +205,49 @@ def hostname_format_valid?
 
   default_value_for :enabled, true
 
+  after_save :change_maintenance_for_child_managers, :if => proc { |ems| ems.changed_attributes.include?('enabled') }
+
+  def disable!
+    _log.info("Disabling EMS [#{name}] id [#{id}].")
+    update!(:enabled => false)
+    _log.info("Disabling EMS [#{name}] id [#{id}] successful.")
+  end
+
+  def enable!
+    _log.info("Enabling EMS [#{name}] id [#{id}].")
+    update!(:enabled => true)
+    _log.info("Enabling EMS [#{name}] id [#{id}] successful.")
+  end
+
+  # Move ems to maintenance zone and backup current one
+  def pause!
+    _log.info("Pausing EMS [#{name}] id [#{id}].")
+    update!(
+      :zone_before_pause => zone,
+      :zone              => Zone.maintenance_zone,
+      :enabled           => false
```

Comment:
Should this instead call the `disable!` method rather than updating the attribute?  Or is the `disable!` method unnecessary?

Reply:
`disable!` method updates only `enabled => false`. 
Disabled method is necessary due to:
1) actually used by OpenShift's Disable button (former "suspend" provider)
2) used by disabling child providers (full pausing unwanted by some reviewer here)

Reply:
> 2\. full pausing unwanted by some reviewer here)

What was the reasoning?  Sorry, I can't find it with all of this comments.

It seems dangerous to pause the provider in manageiq but not it's managers.

Reply:
> It seems dangerous to pause the provider in manageiq but not it's managers.

Not to be confused with `Providers`, these are `ExtManagementSystems` that have child `ExtManagementSystems`

Reply:
Thanks @bdunne, I often mix the terms.  Naming is hard.

@slemrmartin I can see what it does... I'm asking "why" we do not disable the child managers when we disable the parent.  I can't find that discussion in this long list of comments.  I'm hoping you know why.  Thanks.

Reply:
Yeah I think having the "main" EMS being in maintenance zone and its child_emss being in their original zones is a big problem.  The child managers/emss almost always talk to the same system as the primary ems so pausing one without the other doesn't make much sense.

Now I could see pausing a child manager and not its parent (aka the kubevirt InfraMananger child of an OpenShift ContainerManager), that is a valid use case but I think that already works based on this implementation.

Reply:
@agrare @jrafanie child managers are disabled (enable: false) here, moving child manager's zone to same zone as parent ensures parent (it's already implemented), see comment https://github.com/ManageIQ/manageiq/pull/17452#issuecomment-432958273

Reply:
Now the child managers are being paused/resumed https://github.com/ManageIQ/manageiq/pull/17452/files#diff-bc914c1dea04b4ba49055f79a8ab8b5aR813, resolving